addons/test_mail/security/ir.model.access.csv

```diff
@@ -46,3 +46,4 @@ access_mail_test_track_compute,mail.test.track.compute,model_mail_test_track_com
 access_mail_test_track_monetary,mail.test.track.monetary,model_mail_test_track_monetary,base.group_user,1,1,1,1
 access_mail_test_track_selection_portal,mail.test.track.selection.portal,model_mail_test_track_selection,base.group_portal,0,0,0,0
 access_mail_test_track_selection_user,mail.test.track.selection.user,model_mail_test_track_selection,base.group_user,1,1,1,1
+access_mail_test_properties_all,mail.test.properties.user,model_mail_test_properties,,1,1,1,1
```

Comment:
Even if that's a test model, I would make real rules :) (aka: internal only by default)